flocker/provision/_install.py

```diff
@@ -11,12 +11,27 @@
 from urlparse import urljoin
 from characteristic import attributes
 
+from flocker import __version__ as flocker_version
+from flocker.docs import get_doc_version, is_release
 from ._common import PackageSource
 
 ZFS_REPO = ("https://s3.amazonaws.com/archive.zfsonlinux.org/"
             "fedora/zfs-release$(rpm -E %dist).noarch.rpm")
-CLUSTERHQ_REPO = ("https://storage.googleapis.com/archive.clusterhq.com/"
-                  "fedora/clusterhq-release$(rpm -E %dist).noarch.rpm")
+
+if is_release(get_doc_version(flocker_version)):
+    key = 'marketing'
+else:
+    key = 'development'
```

Comment:
We should have separate repositories. But we should only have a single `clusterhq-release` package (that has configuration for both of them). We probably won't have any extra packages that matter for acceptance testing in the development repo (if we do, the work in #1181 can perhaps be used to support it).

That is, we will upload the RPMs to separate repositories, but since we aren't installing those RPMs anyway, we don't need to care about the distinction in `flocker.provision`.